updated PR (#651)
CodeYourFuture/curriculum: .github/pull_request_template.md

## What does this change?

<!-- Add a description of what your PR changes here -->

### Common Content?

> <!-- This PR adds content to the common-content module -->

Reviewer: ```suggestion
<!-- Does this PR adds content to the common-content module? -->
```

> Block/s

Reviewer: ```suggestion
- [ ] Block/s
```

### Common Theme?

> <!-- This PR adds a feature or bugfix to the common-theme module.
> Please reference the ticket you are addressing .-->
> 
> Issue number: #issue-number

Reviewer: ```suggestion
<!-- Does this PR add a feature or bugfix to the common-theme module? -->

- [ ] Yes

<!--Please reference the ticket you are addressing -->

Issue number: #issue-number
```

### Org Content?

> <!--This PR changes a whole module, a sprint, a page, or a block on a single organisation's module.  -->

Reviewer: ```suggestion
<!-- Does this PR changes a whole module, a sprint, a page, or a block on a single organisation's module? -->
```

> Module | Sprint | Page Type | Block Type

Reviewer: ```suggestion
- [ ] Module
- [ ] Sprint
- [ ] Page Type
- [ ] Block Type
```

Author: No tbh I want them to write what it's changing, not checklist it, just so I know where to look

Reviewer: Fair 👍🏻
